src/main/java/billgates/usecases/user_join/UserJoinUseCase.java

```diff
@@ -0,0 +1,29 @@
+package billgates.usecases.user_join;
+
+import billgates.database.QueryUserData;
+import billgates.interface_adapters.DatabaseGateway;
+
+public class UserJoinUseCase implements UserJoinInputPort {
+    private final DatabaseGateway gateway;
+    private final UserJoinOutputPort outputPort;
+
+    public UserJoinUseCase(DatabaseGateway gateway, UserJoinOutputPort outputPort){
+        this.gateway = gateway;
+        this.outputPort = outputPort;
+    }
+
+    @Override
+    public void join(UserJoinRequestModel model) {
+        QueryUserData userData = gateway.getUserData();
```

Comment:
![image](https://user-images.githubusercontent.com/60980117/202885634-03d9a77f-cf00-4b8d-b6de-53f5847f1bd8.png)
According to ray's new pull request, this should be a list of `QueryUserData`

Reply:
Should I code according to @Lei-Tin 's new commit and leave the rest? Would the auto-grading show errors?

Reply:
You could leave your code like this right now. And we could merge your pull request first, then you could modify it after merging Ray's pull request.

Reply:
But you have to fix the other problems that I mentioned before we merge your pull request.